src/components/scheduling/SchedulingEditor.svelte

```diff
@@ -28,11 +38,28 @@
     typescriptDefaults.setCompilerOptions({ ...options, lib: ['esnext'], strictNullChecks: true });
     typescriptDefaults.setExtraLibs(schedulingTsFiles);
   }
+
+  function onSelectReferenceModel(event: CustomEvent<SelectedDropdownOptionValue>) {
+    const { detail: modelId } = event;
+    dispatch('selectReferenceModel', modelId);
+  }
 </script>
 
 <Panel overflowYBody="hidden">
   <svelte:fragment slot="header">
     <SectionTitle>{title}</SectionTitle>
+    <div class="dropdown-select">
+      {#if !readOnly}
+        <label for="models">Reference Model:</label>
+        <SearchableDropdown
```

Comment:
Is it expected that these errors will appear? Seeing this after fresh build on the required backend branch.
![image](https://github.com/NASA-AMMOS/aerie-ui/assets/4419607/d545ffac-0e0f-41b6-9103-174eed669089)

Reply:
@AaronPlave nice catch! I was fetching the wrong TS files. Should be fixed now!